app/src/test/java/com/eis/phoneringer/structure/RingCommandHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package com.eis.phoneringer.structure;
+
+import com.eis.smslibrary.SMSPeer;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Unit testing of the class RingCommandHandler
+ *
+ * @author Alberto Ursino
+ */
+public class RingCommandHandlerTest {
+
+    private static final String SPLIT_CHARACTER = "_";
```

Comment:
You could replace it with
`SPLIT_CHARACTER  = RingCommandHandler.SPLIT_CHARACTER` so that if it changes it's always updated